js/widgets/toolbar.js

```diff
@@ -21,7 +21,8 @@ define( [
 			theme: null,
 			addBackBtn: false,
 			backBtnTheme: null,
-			backBtnText: "Back"
+			backBtnText: "Back",
+			classes: {}
```

Comment:
Remove this even though i told you to do it the widget factory adds the option now in the base widget